system/schema.tql

```diff
@@ -10,12 +10,15 @@ define
     entity user,
         owns uuid @unique @card(1),
         owns name @unique @card(1),
-        plays user-password:user;
+        plays user-credentials:user;
 
-    entity password,
-        owns hash @card(1),
-        plays user-password:password;
+    entity credentials,
+        owns uuid @card(1),
+        plays user-credentials:credentials;
 
-    relation user-password,
+    entity password, sub credentials,
+        owns hash @card(1);
+
+    relation user-credentials,
         relates user @card(1),
-        relates password @card(1);
\ No newline at end of file
+        relates credentials @card(1..);
```

Comment:
Let's add an empty line at the end of the file while we are here

Reply:
Fixed in f4dca852654fa7d18d662be1ccb685a02d3b8a4d.